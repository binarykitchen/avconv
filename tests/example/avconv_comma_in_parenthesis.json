{
  "input":{
    "duration":56100,
    "start":0,
    "bitrate":230,
    "stream":[
      [
        {
          "type":"video",
          "codec":"h264",
          "format":"yuv420p",
          "resolution":[
            854,
            480
          ],
          "bitrate":99,
          "fps":25.02
        },
        {
          "type":"audio",
          "codec":"aac",
          "samplerate":44100,
          "channels":2,
          "sampleformat":"fltp",
          "bitrate":125
        }
      ]
    ]
  },
  "output":{
    "stream":[
      [
        {
          "type":"video",
          "codec":"h264",
          "format":"yuv420p",
          "resolution":[
            854,
            480
          ],
          "bitrate":256
        },
        {
          "type":"audio",
          "codec":"aac",
          "samplerate":44100,
          "channels":2,
          "sampleformat":"fltp",
          "bitrate":128
        }
      ]
    ]
  }
}
