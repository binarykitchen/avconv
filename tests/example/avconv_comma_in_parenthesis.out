ffmpeg version 3.2.14-1~deb9u1 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 6.3.0 (Debian 6.3.0-18+deb9u1) 20170516
  configuration: --prefix=/usr --extra-version='1~deb9u1' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --enable-gpl --disable-stripping --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libebur128 --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmp3lame --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-omx --enable-openal --enable-opengl --enable-sdl2 --enable-libdc1394 --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libopencv --enable-libx264 --enable-shared
  libavutil      55. 34.101 / 55. 34.101
  libavcodec     57. 64.101 / 57. 64.101
  libavformat    57. 56.101 / 57. 56.101
  libavdevice    57.  1.100 / 57.  1.100
  libavfilter     6. 65.100 /  6. 65.100
  libavresample   3.  1.  0 /  3.  1.  0
  libswscale      4.  2.100 /  4.  2.100
  libswresample   2.  3.100 /  2.  3.100
  libpostproc    54.  1.100 / 54.  1.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'evoca venditalia amiclo 1024x576.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf56.40.101
  Duration: 00:00:56.10, start: 0.000000, bitrate: 230 kb/s
    Stream #0:0(und): Video: h264 (Main) (avc1 / 0x31637661), yuv420p(tv, bt470bg/bt470bg/unknown), 854x480 [SAR 1:1 DAR 427:240], 99 kb/s, 25.02 fps, 25 tbr, 90k tbn, 50 tbc (default)
    Metadata:
      handler_name    : VideoHandler
    Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 125 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
[libx264 @ 0x5597be1e3700] using SAR=1/1
[libx264 @ 0x5597be1e3700] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 0x5597be1e3700] profile High, level 3.0
[libx264 @ 0x5597be1e3700] 264 - core 148 r2748 97eaef2 - H.264/MPEG-4 AVC codec - Copyleft 2003-2016 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=-2 threads=12 lookahead_threads=2 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=abr mbtree=1 bitrate=256 ratetol=1.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
Output #0, mp4, to 'out.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf57.56.101
    Stream #0:0(und): Video: h264 (libx264) ([33][0][0][0] / 0x0021), yuv420p, 854x480 [SAR 1:1 DAR 427:240], q=-1--1, 256 kb/s, 25 fps, 12800 tbn, 25 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc57.64.101 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/256000 buffer size: 0 vbv_delay: -1
    Stream #0:1(und): Audio: aac (LC) ([64][0][0][0] / 0x0040), 44100 Hz, stereo, fltp, 128 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
      encoder         : Lavc57.64.101 aac
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> h264 (libx264))
  Stream #0:1 -> #0:1 (aac (native) -> aac (native))
Press [q] to stop, [?] for help
frame= 1401 fps=469 q=-1.0 Lsize=    1837kB time=00:00:56.09 bitrate= 268.2kbits/s speed=18.8x    
video:1776kB audio:14kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 2.556394%
[libx264 @ 0x5597be1e3700] frame I:11    Avg QP: 4.98  size:  7260
[libx264 @ 0x5597be1e3700] frame P:522   Avg QP: 7.35  size:  2816
[libx264 @ 0x5597be1e3700] frame B:868   Avg QP: 7.56  size:   310
[libx264 @ 0x5597be1e3700] consecutive B-frames: 14.6%  5.7%  7.7% 71.9%
[libx264 @ 0x5597be1e3700] mb I  I16..4: 76.9% 15.1%  8.1%
[libx264 @ 0x5597be1e3700] mb P  I16..4:  3.0%  1.6%  0.9%  P16..4:  6.2%  1.5%  0.9%  0.0%  0.0%    skip:86.0%
[libx264 @ 0x5597be1e3700] mb B  I16..4:  0.1%  0.1%  0.1%  B16..8:  2.4%  0.2%  0.1%  direct: 0.5%  skip:96.5%  L0:47.4% L1:48.4% BI: 4.2%
[libx264 @ 0x5597be1e3700] final ratefactor: 13.36
[libx264 @ 0x5597be1e3700] 8x8 transform intra:25.0% inter:20.1%
[libx264 @ 0x5597be1e3700] coded y,uvDC,uvAC intra: 19.5% 27.8% 17.9% inter: 1.4% 2.1% 1.4%
[libx264 @ 0x5597be1e3700] i16 v,h,dc,p: 68% 18%  5%  8%
[libx264 @ 0x5597be1e3700] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 35% 12% 48%  1%  1%  1%  1%  1%  1%
[libx264 @ 0x5597be1e3700] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 35% 24% 19%  3%  4%  4%  3%  5%  3%
[libx264 @ 0x5597be1e3700] i8c dc,h,v,p: 69% 14% 14%  3%
[libx264 @ 0x5597be1e3700] Weighted P-Frames: Y:11.9% UV:5.0%
[libx264 @ 0x5597be1e3700] ref P L0: 75.4%  4.1% 18.1%  2.0%  0.4%
[libx264 @ 0x5597be1e3700] ref B L0: 84.3% 14.1%  1.6%
[libx264 @ 0x5597be1e3700] ref B L1: 98.1%  1.9%
[libx264 @ 0x5597be1e3700] kb/s:259.59
[aac @ 0x5597be237760] Qavg: 65536.000
